backend/backend/src/main/java/de/aaaaaaah/velcom/backend/restapi/exception/ArgumentParseExceptionMapper.java

```diff
@@ -0,0 +1,30 @@
+package de.aaaaaaah.velcom.backend.restapi.exception;
+
+import de.aaaaaaah.velcom.backend.restapi.endpoints.utils.ArgumentParseException;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.Response.Status;
+import javax.ws.rs.ext.ExceptionMapper;
+
+public class ArgumentParseExceptionMapper implements ExceptionMapper<ArgumentParseException> {
+
+	@Override
+	public Response toResponse(ArgumentParseException exception) {
+		return Response
+			.status(Status.FORBIDDEN)
```

Comment:
Why is this a 403 and not just a [400](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/400)?
> The HyperText Transfer Protocol (HTTP) 400 Bad Request response status code indicates that the server cannot or will not process the request due to something that is perceived to be a client error (e.g., malformed request syntax, invalid request message framing, or deceptive request routing).

> The client should not repeat this request without modification.

A 403 implies the request was correct, but the server refused to comply because the user lacked necessary permissions:
> The HTTP 403 Forbidden client error status response code indicates that the server understood the request but refuses to authorize it.